.env.production

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_FF_API_URL=https://api-prod.communityfridgefinder.com
```

Comment:
Maybe for a separate ticket but can we add api-dev as well? 

Since we're live now, we should do a 2 step deploy. Go to dev/staging and then deploy to prod

Reply:
You want the code deployed to https://dev.d1zqbru9k8x5eq.amplifyapp.com/ to send POST/GET to https://api-dev.communityfridgefinder.com?